### DISCUSSION TITLE
Documentation for Solr Indexing

### DISCUSSION THREAD

**Discussion by Aashay-Chapatwala** [DigitalPebble/storm-crawler#1031] [Q&A]
Hi Team,

we are trying to index Solr through  Storm crawler on my local machine. Is there any official documentation for this.

Issue we are facing is, It is not able to crawl on the websites which I mentioned on crawler.flux file and also when I am trying to execute the crawler jar through cmd it is not giving any error as well only just my topology is getting submitted by Storm Submitter on Apache Storm as shown below :

**_Finished submitting topology: mc_crawler_v1_**

FYI :

Detailed logs I have attached in this mail.
Storm crawler version : 2.4
Apache Storm Version : 2.4.0
Apache Solr Version : 8.8.1
Java version : 11
Maven version : 3.5.3

**Comment by jnioche** [comment]
can you share the command you used and put the logs in attachment? thanks
